app/controllers/jobs_controller.rb

```diff
@@ -7,5 +7,7 @@ def index
 
   def show
     @job = Job.find(params[:id])
+    @job_description =
```

Comment:
What is going on here? Should we be formatting the information better before it hits the DB?